src/docs/user/references/Notification-service.adoc

```diff
@@ -0,0 +1,237 @@
+=== Overview
+
+The <<_glossary_notification_portal,*Notification Portal*>> allows users to subscribe, be notified and react when different
+types of events occur during <<_glossary_job,*Job*>> execution, on the state of the <<_glossary_proactive_scheduler,*ProActive Scheduler*>> or when
+automating and scheduling jobs with the <<_glossary_job_planner,*Job Planner*>>.
+
+image::notifications-view.png[align="center"]
+
+=== Subscriptions
+
+==== Principle
+
+In order to be notified of events a user must have an active <<_glossary_notification_subscription,*Subscription*>> with an associated active notification method.
+It is possible to be notified via the <<_glossary_notification_portal,*Notification Portal*>>, email
+or <<_glossary_notification_third_party,*Third party*>> which is the execution of a script provided by the user.
+
+All these notification methods can be used simultaneously in the same <<_glossary_notification_subscription,*Subscription*>>.
+
+==== Subscription types
+
+There are three types of <<_glossary_notification_subscription,*Subscriptions*>>:
```

Comment:
Actually there are two types of subscriptions:
 - Service subscriptions (Scheduler, Job Planner).
 - Job subscriptions

Service subscriptions are unique for each service. They can be enabled or disabled for each particular service.

Job Subscriptions can be multiple, one can create various subscriptions for various kind of workflow types.